testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_replication_filtering.py

```diff
@@ -0,0 +1,585 @@
+import time
+
+import pytest
+import random
+
+from keywords.MobileRestClient import MobileRestClient
+from keywords.utils import add_new_fields_to_doc
+from keywords.utils import log_info
+from CBLClient.Database import Database
+from CBLClient.Replication import Replication
+from libraries.testkit import cluster
+
+
+@pytest.fixture(scope="function")
+def setup_teardown_test(params_from_base_test_setup):
```

Comment:
looks like this set up is not used in any of the test, you can remove this

Reply:
removed